### ISSUE TITLE
Dedicated Google Drive (UW) storage for BHTOM

### ISSUE THREAD

**Issue by astromiki** [BHTOM-Team/bhtom2#115]
Aquire @uw.edu.pl e-mail account for BHTOM in order to set up FITS processing pipeline using OAuth2 protocol. This should be aimed for less interactive users (like Josch Hambsch).

**Comment by wyrzykow** [comment]
Good idea. Expand on it, @astromiki How much disk space we would need for this? 
Since BHTOM moves away from UW next year, we should look for other options, unless we get some other funding within UW (e.g. Impress-U). 
How safe would that be? Do we give access to the drive only to people who ask for it, or to anyone? Do we process every file uploaded there? 
Soon we will not be processing files per-target, but just generally, obtaining photometry for all objects detectable on the frame, it will simplify things - we will not require the target to be known and associated to a fits file.

**Comment by astromiki** [comment]
We should have at least 4TB of space, ideally 8 TB to have copy of a copy. Buying two external 4TB disk is about 1k PLN. Buying cloud space is about the same, but has to be renewed. I would go for the first option. Disk may be installed wherever, f.e. in Wroclaw where we already have the infrastructure suitable to handle this task. I am attaching @kkotysz to this issue.

**Comment by wyrzykow** [comment]
Google Drive is a good idea as the mean to transfer data. 
However, the storage should be solved separately. We will be buying computers, one for OAUW, one for NCBJ, which could both act for storage of the already processed files. They could have the Google Drive mounted and keep all the files which need to be processed, and files which were processed.

**Comment by astromiki** [comment]
I have tested the rclone protocol for downloading and uploading files to GDrive for ROAD data. It works very well and fast. Storing some data could therefore be possible with GDrive, but the biggest upside is that we can easily download data directly from users' GDrives and process it with BHTOM/CCDPhot. It would make the work of many people much easier!
@wyrzykow I will let you know here that automatic ROAD processing is finished.
